src/WebJobs.Script.WebHost/Middleware/JobHostCorsMiddleware.cs

```diff
@@ -24,7 +25,7 @@ public async Task Invoke(HttpContext context, RequestDelegate next)
         {
             context.Items.Add(ScriptConstants.CorsMiddlewareRequestDelegate, next);
 
-            await _corsMiddleware.Invoke(context);
+            await _corsMiddleware.Invoke(context, context.RequestServices.GetService<ICorsPolicyProvider>());
```

Comment:
@fabiocav -- this changed in 3.0. Look okay? I'm not sure if this service is auto-registered or not...